guides/common/modules/proc_importing-kickstart-repositories.adoc

```diff
@@ -5,7 +5,7 @@ Kickstart repositories are not provided by the Content ISO image. To use Kicksta
 
 .Procedure
 
-. Navigate to the Red{nbsp}Hat Customer Portal at https://access.redhat.com/ and log in.
+. Log in to the Red Hat Customer Portal at https://access.redhat.com.
```

Comment:
```suggestion
. Log on to the Red Hat Customer Portal at https://access.redhat.com.
```

Reply:
Docs always say log on, unless referring to the specific UI element? This would break with wider consistency.